www/_posts/2021-08-21-removal-of-translated-docs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Notice of the Removal of the Translated Docs"
+categories: announcements
+tags: news
+---
+
+We like to announce that we have deprecated and removed all translated docs.
+
+We had been providing translated docs since Cordova CLI 3.1.0 but, the number of resources has decreased to a point where it affects our ability to maintain them. The maintenance of these translated docs started to diverge around Cordova CLI 6.x.
```

Comment:
When you say "the number of resources", do you mean "the number of resources required to maintain them"?

Reply:
`the number of resources` has decreased to a point where it affects our ability to `maintain them`.

Resources = people